langchain/agents/load_tools.py

```diff
@@ -124,9 +124,15 @@ def _get_wolfram_alpha(**kwargs: Any) -> Tool:
 
 
 def _get_google_search(**kwargs: Any) -> Tool:
+    if "results" in kwargs:
```

Comment:
thoughts on just making this a separate tool?

Reply:
Hmm.. GoogleSearchAPIWrapper, just seems like both `run` and `results` are available in the wrapper, so they should both be available via `load_tools` as well.

Reply:
agree they should be available, but (1) it may be cleaner from a code perspective to have them separate, (2) having them separate would allow to have both of them usable in same go (may want to use them for different things), (3) depending on whether using results or not they should have different descriptions right? detailing what the output is

Reply:
I don't feel strongly either way. But yeah, i supposed if you're gettin the JSON back then the default description would need to be different to let the agent know that.

Reply:
imo should be separate. at the same time, we're going to be doing a big push on tools next week - so maybe worth saving for then